tango_simlib/examples/weather3.py

```diff
@@ -4,11 +4,10 @@
 
 
 def main():
-    sim_data_files = ['../tests/weather_sim.xmi', '../tests/weather_SIMDD_3.json']
+    sim_data_files = ['../tests/weather.xmi', '../tests/Weather_SIMDD_3.json']
```

Comment:
I think you need a capital "W" for the first one as well.  The file is `Weather.xmi`.